src/Extensions/Logging/Internal/EventSource/OmexLogEventSender.cs

```diff
@@ -0,0 +1,182 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Logging.Replayable;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal sealed class OmexLogEventSender : ILogEventSender, ILogEventReplayer
+	{
+		static OmexLogEventSender()
+		{
+			Process process = Process.GetCurrentProcess();
+			s_processName = string.Format(CultureInfo.InvariantCulture, "{0} (0x{1:X4})", process.ProcessName, process.Id);
+		}
+
+
+		public OmexLogEventSender(OmexLogEventSource eventSource, IMachineInformation machineInformation, IServiceContext context)
+		{
+			m_eventSource = eventSource;
+			m_machineInformation = machineInformation;
+			m_serviceContext = context;
+		}
+
+
+		public void LogMessage(string activityId, ActivityTraceId traceId, string category, LogLevel level, EventId eventId, int threadId, string message)
+		{
+			if (!IsEnabled(level))
+			{
+				return;
+			}
+
+			Guid partitionId = m_serviceContext.PartitionId;
+			long replicaId = m_serviceContext.ReplicaOrInstanceId;
+			string applicationName = m_machineInformation.MachineRole;
```

Comment:
Can we align these names, preferably by changing `MachineRole` to `applicationName`

Reply:
Renamed to `ApplicationName`, thank you